solutions/gold/cses-1095.mdx

```diff
@@ -56,4 +56,31 @@ int main() {
 }
 ```
 </CPPSection>
+
+<PySection>
+```py
+import sys
+
+input = sys.stdin.readline
+takeInt = lambda: int(input())
+takeInts = lambda: map(int, input().split())
+takeList = lambda: list(map(int, input().split()))
```

Comment:
remove these, you only call them once anyways.

Reply:
ok i will change them. please give me the clarity do i use the inbuilt pow function or my custom function

Reply:
use the builtin

Reply:
builtin might tle

Reply:
can't we include both as two solutions?

Reply:
in the module, perhaps, but this solution should definitely use the builtin